summaries/2023-03-03.md

```diff
@@ -0,0 +1,26 @@
+# MOB Programming Session
+
+Date: 03-03-2023
+
+Topic: Array methods - Map, Filter, Reduce
+
+# Related Documentation
+
+> [JavaScript Map, Reduce, and Filter](https://www.freecodecamp.org/news/javascript-map-reduce-and-filter-explained-with-examples/)
+
+> [Map, Filter, Reduce - Code Exercises](https://www.crocoder.dev/blog/map-filter-reduce-exercises)
+
+## Summary
+
+Map, reduce, and filter are all array methods in JavaScript. Each one will iterate over an array and perform a transformation or computation. Each will return a new array based on the result of the function.
+
+Continuing from the last MOB session with TDD, we once again wrote unit tests about manipulating an array: doubling the values, taking out the even values and making a sum of all the values in the array.
```

Comment:
Last time we did do TDD, but we didn't handle manipulating an array. Think this can be worded better